include/aspect/particle/world.h

```diff
@@ -347,7 +347,7 @@ namespace aspect
          */
         virtual
         void
-        parse_parameters (ParameterHandler &prm);
+        parse_parameters (ParameterHandler &prm, const unsigned int world_index);
```

Comment:
Add documentation for this parameter.  Also I cant decide right now if it is better to hand over the index, or the constructed subsection name as string. I need to think about this more. What would be your opinion?

Reply:
I think I am in favor of passing the raw information in this case. I think it is a good think that the particle world can be aware what particle world it is, without having to parse a string it got.